### ISSUE TITLE
What usage restrictions do we want to impose and how?

### ISSUE THREAD

**Issue by zspencer** [zinc-collective/compensated#55]
Part of the goal with Compensated (and @zinc-collective in general) is to demonstrate that we can create socioeconomically restorative organizations through cooperatively owned and operated digital products and services.

To do so, we want to be careful about the usage rights we want to grant people and organizations to "curb" some of the most egregious violations of human rights that are currently encouraged due to insufficient regional and national legislation.

While these restrictions will by definition slow adoption of Compensated; we believe that's a reasonable price to pay in order to stop the cycle of exploitation, surveillance, incarceration, and war being used to enrich the few at the cost of the many.

I'm seeing consensus emerge is that _licensing_ is unlikely to be an effective mechanism to restrict usage rights along ethical grounds; as once a license is granted they are difficult (if not impossible) to revoke. As a result; we may want to investigate requiring licensees to agree to terms of use which revoke their license(s).

I propose we restrict the following usage:

1. Child labor
1. Weapon production and distribution
1. Surveillance capitalism
1. War
1. Incarceration
1. Fossil fuel extraction
1. Creation or distribution of hate speech

Thoughts?

**Comment by drllau** [comment]
> I'm seeing consensus emerge is that licensing is unlikely to be an effective mechanism to restrict usage rights along ethical grounds; as once a license is granted they are difficult (if not impossible) to revoke. 

Because you hit the fundamental constraint of contract law (doctrine of [privity](https://en.wikipedia.org/wiki/Privity_of_contract) only direct parties are bound) of which software license is a special subclass over intangible exchange of property (debt based obligation). Hence why I'd advocate Economic [Tort](https://en.wikipedia.org/wiki/Duty_of_care) (harm based damages) which is a more generic concept of neighbour, based on the Christian principle of "love your neighbour as yourself". 

> The duty of care may be imposed by operation of law between individuals who have no current direct relationship (familial or contractual or otherwise) but eventually become related in some manner, as defined by common law (meaning case law)

Many of the objections above (child labor, weapon [proliferation](https://en.wikipedia.org/wiki/Right_to_keep_and_bear_arms), etc) are already governed by law ... **inside** the US but a fundamental principle is sovereignty in that you cannot export your laws to another country (though tried in past like tag jurisdiction ... case of passenger in plane flying over Texas given divorce papers]. That's why I've been trying to steer away from trying to enforce your ethical beliefs via contract, and instead more on cryptoeconomics (game theory), tort of fidelity, and perhaps a touch of trust-based equity as restorative justice (alas which is not recognised by civil authorities).

> As a result; we may want to investigate requiring licensees to agree to terms of use which revoke their license(s).

Moral authority != actual judicial authority. If at any point you want to use the legal system, ultimately you are resorting to coercion (xref your point about incarceration). The best way to win is to make zinc-collective/compensated so prevalent that a **boycott by operators** that affirms your social compact and norms (see Principle 2 = collective action) is economically crippling to the perpetrator, plus the obligatory social opprobrium. Bitcoin is an example of a system which at first appeared to be anonymous ... without people realising every transaction was recorded and in future if anyone can break the security, some drug dealer descendants are going to find the transition from trust-fund baby to homeless vagabond overdue karma.

**Comment by drllau** [comment]
[Digital platforms inquiry - final report.pdf](https://github.com/zinc-collective/compensated/files/4400517/Digital.platforms.inquiry.-.final.report.pdf)
[Surveillance capitalism](https://www.cmo.com.au/article/666454/facebook-criticises-accc-digital-platform-inquiry-final-report-recommendations/) is harder for collective obfuscation (see report above; TLDR is that company larger than country economicwise). Otherwise here would be my [interpretation](https://en.wikipedia.org/wiki/Alice_and_Bob) of the desired end-goals (policies?) of Zinc

> Child labor

Create alice($work), bob($home), Carol($play) as internet payment interface ... ask for proof of age, or certificate of emancipation in the data jurisidiction for alice($work) ... state that amounts <$10k will be normally cleared immediately but over that next day (allow DevOps to do financial crimes invest + report if necessary). Put inbound limits on bob($home) ... in general it would be for refunds ... you don't care how much is transferred internally from work to home nor what it is spent on, but it is a flag if sum ($home=refunds) > sum($work * (1-taxes)). Carol($play) would be their kid so you can allow additional whitelisted source restrictions bob($home) so parents allowed to set a weekly allowance (say $1 x age after sighting birth cert). The main protection is that any uncapped $work internet payment interface needs an age or emancipation check, plus optional work visa if required by law.

> Weapon production and distribution

when you can define weapon I'll give you counterpoint ... basically I find this unenforceable (and amusing in light of your 2nd ammendment). Hence I refer to **tort of fidelity**, anything which deletes, damages or disdirects an internet payment interface is an economic harm and can be punished if negligence is proven (usual laws of natural justice and balance of probabilities apply). Murder would be morally considered equivalent to deleting a payment point though I don't know the borderline case of shooting up a vending machine.

>Surveillance capitalism

Rules in place ... only 1 day worth of transactions kept for US internet payment portals (to ensure next day clearance of funds held up by Fincin reporting). Messaging systems I believe are 6 months and banks 7 years (at least Aussie law) but since you're a straight through processor, you do want to avoid paperwork right ... ? There are things in life which you control, others can influence and rest accept. Surveillance is partly laziness, mostly greed and a side-dish of govt paranoia. Choosing a good browser like [brave](https://privacyaustralia.net/best-secure-browser/), paying for security, and voting are all [doubleplusgood](https://www.urbandictionary.com/define.php?term=doubleplusgood) actions. Best efforts on engineering stability is what you can control.

> War

Define jurisidictions of parties and issue collective embargo ... funds to go to escrow until someone intelligent (not UN) can sort out the mess.

> Incarceration

define as moral equivalent of temporary disablement of internet payment interface. Treat with policies similar to murder.

> Fossil fuel extraction

if you want to implement a carbon tax, please put forward details :-D (did I tell you my day job used to be low-carbon technologies? I'm still a little miffed the Asian Development Bank didn't accept my proposal) ... I'd implement with some sort surcharge + rebate on renewables if you can define which electron is running on your CPU

> Creation or distribution of hate speech

Right of rebuttal + violation of social norms + sin-bining (temporary diversion of payments as restorative justice).

Basically I'm trying to implement what is feasible within what tools are available
- **cultural lore** - norms and community standards, badges
- **common law** - reusing existing legislation to hard bake into alice($work), bob($home), carol($play) as necessary to implement somewhat sensible defaults and ask licensees to be transparent in their defaults too.
- **commercial lures** - a mix of game theory, marginal cost-benefts (fees/rebates) + economic tort to ecourage positive behaviour.

**Comment by zspencer** [comment]
This looks like a legal evaluation; which feels premature. At present, we do not have consensus on:

A) what usage restrictions do we _want_ to impose and

Which would lead to:

B) How would we go about doing it?

Which may lead to:

C) What (if any) legal mechanisms do we have that are enforceable within the markets we're currently participating in?

There are infinite ways to rip apart any legal document in court, but (as I believe you mentioned) the best contracts never enter court because they're mutually beneficial or enforced through social norms.

Regardless of what we choose for C, we will want _some_ mechanism to communicate what usage we _want_ to prohibit. Whether that mechanism is enforceable in every jurisdiction is a challenge that would require teams of lawyers for months or years. If you're interested in the enforcement side of thing formal legal documents, I encourage you to spend your time and considerable legal expertise working with the folks [in the ethical source working group](https://ethicalsource.dev/apply/), which have a variety of professionals (and amateurs) discussing and debating the implementation merits of individual licenses. In particular, the discussion around [what degree of enforceability do we want to consider when it comes to licenses](https://github.com/EthicalSource/ethicalsource.dev/issues/50) would likely benefit from your thoughtful expertise.

I am willing to do the work to create restorative, just socioeconomic digital systems and am well aware our existing socioeconomic systems are designed to discourage such efforts. I'm less willing to spend significant time and energy trying to convince people that these efforts are worthwhile or achievable; because there are infinite ways they could fail; and I want to focus on finding things that may work despite the powers, structures, and norms that are in opposition to such efforts.

**Comment by drllau** [comment]
> I am willing to do the work to create restorative, just socioeconomic digital systems and am well aware our existing socioeconomic systems are designed to discourage such efforts. 

Totally agree ... the point wasn't the legal lingo per se but that it forces clarity of thinking. What rights are created are based on the relationship between the parties (contractually) or group (tort). BTW it's not that socio-economic systems are designed to discourage, its just that systems evolve to a point because of historical antecedents, then tries to maintains stability. It perceives some rebalancing activities as subversive by current elites (think trust-fund babies or overpaid PHB) who then use existing powers to stomp on them ... IBM vs 7 dwarfs, MS vs Netscape, Google vs Sun, iOS store vs every eCommerce app, the list repeats ... 

![Points of Control](https://user-images.githubusercontent.com/14944510/77974940-2339cd00-732b-11ea-877b-93d8e2dd3ec0.png)

All I want is space to go into seclusion with decent mind conversation and not beholden to a platform who wants to waste my time.

**Comment by zspencer** [comment]
I et that. I'm going to keep musin on this and see if other folks have thoughts or insights about what we want to do beyond the restrictions outlined above.
